web/impact/impact/permissions.py

```diff
@@ -73,6 +77,19 @@ def has_permission(self, request, view):
             name=settings.V0_API_GROUP).exists()
 
 
+class DirectoryAccessPermissions(BasePermission):
```

Comment:
It has been an expectation in our code that each class should go in a file named for the class - by that standard, this should be in a file called `directory_access_permissions.py`. However, I see that that standard has been allowed to slip. I also see that there is an AllocateApplicationsPermissions class in v1/classes/. It seems like we should probably make some consistent treatment of these things. 

It seems that the right thing to do here would be to make permissions.py into a directory and extract the random classes into correctly-named files, then collect AllocateApplicationsPermissions into that directory.

Reply:
It looks like the class you mean here `v1/classes/` is inheriting from the `permissions.py`, which leads me to think it may be intentional. No?

Reply:
well, yeah, it's inheriting from permissions.py - but probably should be with the rest of the permissions. 
I'm not going to insist on this cleanup now, but in general it's a lot better to clean up messes as you find them than to try to schedule tickets.

Reply:
Doing it now.

Reply:
Tis done, permissions cleaned up on the directory!

Reply:
on **impact!